pixi.toml

```diff
@@ -126,6 +126,12 @@ ruff = ">=0.4.9"
 pyright = ">=1.1.368"
 
 
+# build
+[feature.dev.dependencies]
+cmake = "3.16.*"
+ninja = "1.12.1"
+make = ">=4.3,<5"
```

Comment:
we should probably make a separate environment set for things that are required for building but not necessarily for development (e.g., if people do want to use pixi for running/building but don't want things like linters)

Reply:
Added a build feature and another build env. I added the build feature and cuda121 and torch121, but idk if those are what you want there - lmk and I can update it.